_data/site_menu.yml

```diff
@@ -31,6 +31,13 @@ installation:
       path: "/docs/reference/latest/installing_on_scvmm/index.html"
       desc: "How to install and configure the ManageIQ Management Engine appliance on Microsoft System Center Virtual Machine Manager (SCVMM)"
 
+appliance_backup:
```

Comment:
Should appliance backup and restore live next to appliance upgrade?

Reply:
Yes, maybe perhaps under a separate sub-menu?